doc/lua_api.txt

```diff
@@ -3943,7 +3943,9 @@ Definition tables
         diggable = true, -- If false, can never be dug
         climbable = false, -- If true, can be climbed on (ladder)
         buildable_to = false, -- If true, placed nodes can replace this node
-        floodable = false, -- If true, liquids flow into and replace this node
+        floodable = false, --[[
+        ^ If true, liquids flow into and replace this node. However, liquids
+        ^ cannot themselves be floodable. ]]
```

Comment:
Liquids can be made floodable, it's just a very bad idea. So this needs to be clearer.
`If true, liquids flow into and replace this node`
newline
`Warning: Making liquid nodes floodable causes an extreme load on waterflow processing`